src/main/java/io/openliberty/tools/intellij/lsp4mp/MicroProfileDeploymentSupport.java

```diff
@@ -38,7 +37,7 @@ public class MicroProfileDeploymentSupport implements ClasspathResourceChangedMa
     private final Project project;
 
     public static MicroProfileDeploymentSupport getInstance(Project project) {
-        return ServiceManager.getService(project, MicroProfileDeploymentSupport.class);

```

Comment:
Definition of  ServiceManager.getService() method.
  
  /*Deprecated
Use ComponentManager. getService(Class): Application. getService() / Project. getService(). */
  @Deprecated
  public static <T> T getService(@NotNull Project project, @NotNull Class<T> serviceClass) {
    return project.getService(serviceClass);
  }

Reply:
Thanks for providing this here.